app/javascript/pages/howToApply/how-to-apply.tsx

```diff
@@ -125,7 +129,38 @@ const eligibilityListItems = [
   { index: 5 },
 ]
 
-const BeforeYouStartSection = () => {
+const LeasingAgentBox = ({ listing }: { listing: RailsSaleListing }) => {
+  return (
+    <div className="leasing-agent">
+      <p className="m-0">{listing.Leasing_Agent_Name}</p>
+      <p className="text-gray-700 text-sm">{listing.Leasing_Agent_Title}</p>
```

Comment:
issue: Fields coming from salesforce are not being machine translated. We will need something similar to [DAH-2623](https://sfgovdt.jira.com/browse/DAH-2623).

Reply:
We probably don't want to translate the Leasing Agent Name?

For `Leasing_Agent_Title` the translations for that field should be on the translations object so we just need to do:
```
getTranslatedString(
                    listing.Leasing_Agent_Title,
                    "Leasing_Agent_Title__c",
                    listing.translations
                  )
```

Reply:
I agree. Name we dont want to translate, `Leasing_Agent_Title` and `Office_Hours` we do. I've updated the ACs on the ticket as well.